wordpress-mobile/WordPress-FluxC-Android
# Removes: blaze status api implementation and data base classes

## Description 
part of https://github.com/wordpress-mobile/WordPress-Android/issues/18704

## TO test
1. Test this change with WP Android PR - https://github.com/wordpress-mobile/WordPress-Android/pull/18718
2. Make sure that the changes for adding the blaze status in this [PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2657/commits) is reverted -
